addons/binding/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/handler/HDPowerViewHubHandler.java

```diff
@@ -58,31 +66,44 @@
     private HDPowerViewWebTargets webTargets;
     private ScheduledFuture<?> pollFuture;
 
+    private final ChannelTypeUID sceneChannelTypeUID = new ChannelTypeUID(HDPowerViewBindingConstants.BINDING_ID,
+            HDPowerViewBindingConstants.CHANNELTYPE_SCENE_ACTIVATE);
+
     public HDPowerViewHubHandler(Bridge bridge) {
         super(bridge);
     }
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-
+        Channel channel = getThing().getChannel(channelUID.getId());
+        if (channel != null && sceneChannelTypeUID.equals(channel.getChannelTypeUID())) {
+            if (command.equals(OnOffType.ON)) {
+                try {
+                    webTargets.activateScene(Integer.parseInt(channelUID.getId()));
+                } finally {
+                    // 5 seconds later, toggle it back off - scenes are stateless
```

Comment:
Err, better don't do this but rather treat it as a known limitation or ask people to add `{ autoupdate="false" }` to their items. The proper solution is to help getting https://github.com/eclipse/smarthome/issues/595 implemented!